metadata-ingestion/tests/integration/metabase/metabase_mces_golden.json

```diff
@@ -180,10 +183,55 @@
                         "customProperties": {},
                         "title": "Dashboard 1",
                         "description": "",
-                        "charts": [
-                            "urn:li:chart:(metabase,1)",
-                            "urn:li:chart:(metabase,2)"
+                        "charts": [],
```

Comment:
seems like the relationships between dashboards and charts disappeared? Given that it's an important part of the integration, can we update the code / test data to make sure we're testing the dashboard -> chart relationship generation?

Reply:
Fixed by:
- https://github.com/datahub-project/datahub/pull/9631/commits/58c65939d04eeae413a8e4f1e90a03c7e1fef099
- https://github.com/datahub-project/datahub/pull/9631/commits/b6150b4c21a49efe2580f0251e23e79277920b40
- https://github.com/datahub-project/datahub/pull/9631/commits/34cdf5541dd874e4166119c6c0f93acc791c29fa